docs/content/users/install/performance.md

```diff
@@ -1,136 +1,145 @@
+
+    While Mutagen works fine and has automated tests for Linux and Windows WSL2, it may not be worth enabling on those systems since it won’t make the dramatic difference it does on macOS and Windows.
     
-    Another major advantage of Mutagen over NFS is that it supports filesystem notifications, so file-watchers on both the host and inside the container will be notified when changes occur. This is a great advantage for many development tools, which had to poll for changes in the past, but now will be notified via normal inotify/fsnotify techniques.
+    Another major advantage of Mutagen over NFS is that it supports filesystem notifications, so file-watchers on both the host and inside the container will be notified when changes occur. This is a great advantage for many development tools, which otherwise have to poll for changes at greater expense. Instead, they can be notified via normal `inotify`/`fsnotify` techniques.
     
-    If you trouble with the Mutagen feature, please try to recreate it and report via one of the [support channels](../support.md).
+    ### Enabling and Disabling Mutagen
 
-    ### Enabling Mutagen
+    !!!warning "Don’t Install Mutagen"
+        Don’t install the `mutagen` binary separately. If it’s not available, DDEV will install and upgrade it for you.
 
-    !!!warning "Do not separately install mutagen"
+    We recommend enabling Mutagen globally with `ddev config global --mutagen-enabled`. You can disable it again with `ddev mutagen reset && ddev config global --mutagen-enabled=false`.
 
-        Do not separately install the mutagen binary. It's better if you don't have it installed. DDEV does the installation and upgrades when needed.
+    If you’d rather use Mutagen on a specific project, run `ddev stop`, enable it with `ddev config --mutagen-enabled`, and `ddev start` again.
 
-    To begin using Mutagen, just `ddev stop` and then `ddev config --mutagen-enabled` and start the project again. If the mutagen binary needs to be downloaded, it will be downloaded automatically.
-    
-    To stop using Mutagen on a project, `ddev mutagen reset && ddev config --mutagen-enabled=false`.
+    You can’t disable Mutagen on individual projects if it’s enabled globally.
     
-    You can also enable mutagen globally (recommended) with `ddev config global --mutagen-enabled`
+    To stop using Mutagen on a project, run `ddev mutagen reset && ddev config --mutagen-enabled=false`.
     
-    Note that the nfs-mount-enabled feature is automatically turned off if you're using mutagen.
-
-    You can run mutagen on all your projects, there's no limit. To configure it globally, `ddev config global --mutagen-enabled`, but you cannot disable mutagen on individual projects if it's enabled globally (the global configuration wins).
+    The `nfs-mount-enabled` feature is automatically turned off if you’re using Mutagen.
     
-    ### Windows Mutagen and executable files
+    ### Windows Mutagen and Executable Files
```

Comment:
This is now fixed, so we can remove

```suggestion
```

Reply:
Not to be exceedingly dull, but you’re recommending we remove the entire section and not just the heading, right?